README.md

```diff
@@ -32,15 +44,26 @@ docker rm -f ubuntu_14.04-2224
 # remove all containers
 docker rm -f $(docker ps -a -q)
 # jump into a container, force a shell
-docker exec -it waffleimage_debian9_-2223 /bin/bash
+docker exec -it litmusimage_debian9_-2223 /bin/bash
 # attach to a container ( limited by what pid 0 is)
-docker attach waffleimage_ubuntu16.04_-2222
+docker attach litmusimage_ubuntu16.04_-2222
 # safely exit a container, leaving it running 
 <ctrl> + p then <ctrl> + q
 # get the latest version of the image
 docker pull debian:8
 # show the history of the image
-docker image history waffleimage/ubuntu16.04
+docker image history litmusimage/ubuntu16.04
 # remove all docker images that are on your local machine
 docker rmi $(docker images -q)
 ```
+
+# Add new images
+
+* Add/change dockerfile for the new image
+* Every dockerfile needs a `base_image` label where the base image id will be stored. This will be used in the nightly build to identify if the base image has been updated.
+* Change workflows at https://github.com/puppetlabs/litmus_image/tree/master/.github/workflows to build the new images
+* New images will be pushed to dockerhub only on pushes to the master branch, and will be updated nightly in case the base image has changed
+
+## Future improvements
+
+* Use a centralized file (`.cvs?`) with the images and coresponding `docker build` parameters rather than having it duplicated in every workflow
```

Comment:
```suggestion
* Use a centralized file (possibly a `.csv`) with the images and coresponding `docker build` parameters rather than having it duplicated in every workflow
```